src/Resources/app/administration/src/module/swag-migration/component/swag-migration-error-resolution/swag-migration-error-resolution-details-modal/swag-migration-error-resolution-details-modal.html.twig

```diff
@@ -0,0 +1,30 @@
+{% block swag_migration_error_resolution_details_modal %}
+    <sw-modal
+        class="swag-migration-error-resolution-details-modal"
+        variant="large"
+        :title="modalTitle"
+        @modal-close="() => $emit('close-error-resolution-details-modal')"
```

Comment:
Does the emitted event need to be this long? Also I think we follow a certain convention for event names with noun + verb 🤔 

```suggestion
        @modal-close="() => $emit('modal-close')"
```